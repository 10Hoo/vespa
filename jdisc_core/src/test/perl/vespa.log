1354022568.091108	example.yahoo.com	18150/38735	configserver	runserver	event	starting/1 name="/home/vespa/bin/vespa-start-container-daemon -Dzookeeper.jmx.log4j.disable=true -Dconfigsources=tcp/localhost:19070 (pid 18151)"
1354022568.100151	example.yahoo.com	18151	configserver	qrs-start	debug	exporting: YELL_MA_EURO=INXIGHT
1354022568.100217	example.yahoo.com	18151	configserver	qrs-start	debug	Not setting ulimit -v, no limit set.
1354022568.120716	example.yahoo.com	18151	configserver	stdout	info	ROOT = /home/vespa
1354022568.232852	example.yahoo.com	18151	configserver	stdout	info	Running without a pid file.
1354022568.336341	example.yahoo.com	18151	configserver	stdout	info	LANG = en_US.UTF-8
1354022568.393706	example.yahoo.com	18151	configserver	stdout	info	env LD_PRELOAD=/home/vespa/libexec64/yjava_daemon_preload.so: /home/vespa/bin64/yjava_daemon -sentinel -rlimit_files 16384 -logdest /home/vespa/logs/vespa/yjava_daemon.out -loglevel error -jvm server -maxrestarts 0 -procs 1 -home /home/vespa/share/yjava_jdk/java -Djava.library.path=/home/vespa/lib64:/home/vespa/lib64 -Dvespa.log.control.dir=/home/vespa/var/db/vespa/logcontrol -XX:ThreadStackSize=512 -XX:+UseConcMarkSweepGC -XX:MaxTenuringThreshold=128 -XX:MaxPermSize=512m -Dconfig.id=dir:/home/vespa/conf/configserver -Dyjava_remote_ip_servlet_filter.logLevel=DEBUG -Dorg.apache.commons.logging.Log=org.apache.commons.logging.impl.Jdk14Logger -Dcom.yahoo.protect.Process.forcedExitActive=true -Dzookeeperlogfile=/home/vespa/logs/vespa/zookeeper.log -Xms1536m -Xmx1536m -XX:MaxDirectMemorySize=267m -Djava.awt.headless=true -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000 -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/home/vespa/var/crash -Dsun.net.client.defaultConnectTimeout=5000 -Dsun.net.client.defaultReadTimeout=60000 -Djavax.net.ssl.keyStoreType=JKS -javaagent:/home/vespa/lib/jars/yjava_ysecure_agent.jar -Djdisc_core.config.file=/home/vespa/var/jdisc_core/config.properties -Djdisc.export.packages=yjava.security.ysecure,yjava.security.yca,yjava.security.yck,org.apache.velocity,org.apache.velocity.app,org.apache.velocity.context,org.apache.velocity.runtime,org.apache.velocity.runtime.log, -Djdisc.cache.path=/home/vespa/var/vespa/bundlecache/dir:_home_y_conf_configserver -Djdisc.debug.resources=false -Djdisc.bundle.path=/home/vespa/lib/jars -Djdisc.logger.enabled=false -Djdisc.logger.level=ALL -Djdisc.logger.tag=dir:/home/vespa/conf/configserver -user yahoo -cp /home/vespa/lib/jars/jdisc_core-with-dependencies.jar:lib/jars/yjava_bcookie.jar:lib/jars/yjava_bcookie_jni.jar:lib/jars/yjava_byauth.jar:lib/jars/yjava_cookie_data_servlet_filter.jar:lib/jars/yjava_daemon.jar:lib/jars/yjava_jmx_singleton_server.jar:lib/jars/yjava_remote_ip_servlet_filter.jar:lib/jars/yjava_resource_handler.jar:lib/jars/yjava_servlet.jar:lib/jars/yjava_servlet_filters.jar:lib/jars/yjava_yca.jar:lib/jars/yjava_yck.jar:lib/jars/yjava_yhdrs:lib/jars/yjava_yiv.jar:lib/jars/yjava_yiv_servlet.jar:lib/jars/yjava_ynet.jar:lib/jars/yjava_ysecure.jar:lib/jars/yjava_ysecure_agent.jar:lib/jars/yjava_ysecure_native.jar:share/jports/org_apache_velocity__velocity.jar:share/jports/commons_collections__commons_collections.jar:share/jports/commons_lang__commons_lang.jar: -ynet FILTER_YAHOO_ANY --wait-for-jvm-init 10 -Dzookeeper.jmx.log4j.disable=true -Dconfigsources=tcp/localhost:19070 -XX:OnOutOfMemoryError="kill -9 %p" com.yahoo.jdisc.core.BootstrapDaemon file:/home/vespa/lib/jars/container-disc-with-dependencies.jar
1354022570.876	example.yahoo.com	18151/1	configserver	Container.com.yahoo.container.handler.config.HandlersConfigurerDi	info	Installing bundles from the latest application
1354022570.884	example.yahoo.com	18151/1	configserver	Container.com.yahoo.container.handler.BundleLoader	info	Installing bundle from disk with reference 'file:/home/vespa/lib/jars/config-models/sherpa-config-model-plugin.jar'
1354022570.889	example.yahoo.com	18151/1	configserver	Container.com.yahoo.container.handler.BundleLoader	info	Installing bundle from disk with reference 'file:/home/vespa/lib/jars/configserver-container-plugin.jar'
1354022574.285	example.yahoo.com	18151/1	configserver	Container.com.yahoo.container.handler.config.HandlersConfigurerDi	info	Installing bundles from the latest application
1354022577.430	example.yahoo.com	18151/1	configserver	Container.com.yahoo.statistics.Counter	event	count/1 name=requests value=0
1354022577.431	example.yahoo.com	18151/1	configserver	Container.com.yahoo.statistics.Counter	event	count/1 name=requestsCached value=0
1354022577.432	example.yahoo.com	18151/1	configserver	Container.com.yahoo.statistics.Counter	event	count/1 name=requestsNotCached value=0
1354022577.432	example.yahoo.com	18151/1	configserver	Container.com.yahoo.statistics.Counter	event	count/1 name=failedRequests value=0
1354022577.432	example.yahoo.com	18151/1	configserver	Container.com.yahoo.statistics.Counter	event	count/1 name=procTime value=0
1354022577.432	example.yahoo.com	18151/1	configserver	Container.com.yahoo.statistics.Counter	event	count/1 name=procTimeCached value=0
1354022577.432	example.yahoo.com	18151/1	configserver	Container.com.yahoo.statistics.Counter	event	count/1 name=procTimeNotCached value=0
1354022577.489	example.yahoo.com	18151/1	configserver	Container.com.yahoo.container.handler.config.HandlersConfigurerDi	info	Switched to the latest deployed set of handlers, and dependent components, e.g. search chains, searchers and document processors. Application switch number: 0
