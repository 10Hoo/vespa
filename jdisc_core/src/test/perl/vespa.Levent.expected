[2012-11-27 14:22:48.120] INFO    : configserver     stdout	ROOT = /home/vespa
[2012-11-27 14:22:48.232] INFO    : configserver     stdout	Running without a pid file.
[2012-11-27 14:22:48.336] INFO    : configserver     stdout	LANG = en_US.UTF-8
[2012-11-27 14:22:48.393] INFO    : configserver     stdout	env LD_PRELOAD=/home/vespa/libexec64/yjava_daemon_preload.so: /home/vespa/bin64/yjava_daemon -sentinel -rlimit_files 16384 -logdest /home/vespa/logs/vespa/yjava_daemon.out -loglevel error -jvm server -maxrestarts 0 -procs 1 -home /home/vespa/share/yjava_jdk/java -Djava.library.path=/home/vespa/lib64:/home/vespa/lib64 -Dvespa.log.control.dir=/home/vespa/var/db/vespa/logcontrol -XX:ThreadStackSize=512 -XX:+UseConcMarkSweepGC -XX:MaxTenuringThreshold=128 -XX:MaxPermSize=512m -Dconfig.id=dir:/home/vespa/conf/configserver -Dyjava_remote_ip_servlet_filter.logLevel=DEBUG -Dorg.apache.commons.logging.Log=org.apache.commons.logging.impl.Jdk14Logger -Dcom.yahoo.protect.Process.forcedExitActive=true -Dzookeeperlogfile=/home/vespa/logs/vespa/zookeeper.log -Xms1536m -Xmx1536m -XX:MaxDirectMemorySize=267m -Djava.awt.headless=true -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000 -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/home/vespa/var/crash -Dsun.net.client.defaultConnectTimeout=5000 -Dsun.net.client.defaultReadTimeout=60000 -Djavax.net.ssl.keyStoreType=JKS -javaagent:/home/vespa/lib/jars/yjava_ysecure_agent.jar -Djdisc_core.config.file=/home/vespa/var/jdisc_core/config.properties -Djdisc.export.packages=yjava.security.ysecure,yjava.security.yca,yjava.security.yck,org.apache.velocity,org.apache.velocity.app,org.apache.velocity.context,org.apache.velocity.runtime,org.apache.velocity.runtime.log, -Djdisc.cache.path=/home/vespa/var/vespa/bundlecache/dir:_home_y_conf_configserver -Djdisc.debug.resources=false -Djdisc.bundle.path=/home/vespa/lib/jars -Djdisc.logger.enabled=false -Djdisc.logger.level=ALL -Djdisc.logger.tag=dir:/home/vespa/conf/configserver -user yahoo -cp /home/vespa/lib/jars/jdisc_core-with-dependencies.jar:lib/jars/yjava_bcookie.jar:lib/jars/yjava_bcookie_jni.jar:lib/jars/yjava_byauth.jar:lib/jars/yjava_cookie_data_servlet_filter.jar:lib/jars/yjava_daemon.jar:lib/jars/yjava_jmx_singleton_server.jar:lib/jars/yjava_remote_ip_servlet_filter.jar:lib/jars/yjava_resource_handler.jar:lib/jars/yjava_servlet.jar:lib/jars/yjava_servlet_filters.jar:lib/jars/yjava_yca.jar:lib/jars/yjava_yck.jar:lib/jars/yjava_yhdrs:lib/jars/yjava_yiv.jar:lib/jars/yjava_yiv_servlet.jar:lib/jars/yjava_ynet.jar:lib/jars/yjava_ysecure.jar:lib/jars/yjava_ysecure_agent.jar:lib/jars/yjava_ysecure_native.jar:share/jports/org_apache_velocity__velocity.jar:share/jports/commons_collections__commons_collections.jar:share/jports/commons_lang__commons_lang.jar: -ynet FILTER_YAHOO_ANY --wait-for-jvm-init 10 -Dzookeeper.jmx.log4j.disable=true -Dconfigsources=tcp/localhost:19070 -XX:OnOutOfMemoryError="kill -9 %p" com.yahoo.jdisc.core.BootstrapDaemon file:/home/vespa/lib/jars/container-disc-with-dependencies.jar
[2012-11-27 14:22:50.876] INFO    : configserver     Container.com.yahoo.container.handler.config.HandlersConfigurerDi	Installing bundles from the latest application
[2012-11-27 14:22:50.884] INFO    : configserver     Container.com.yahoo.container.handler.BundleLoader	Installing bundle from disk with reference 'file:/home/vespa/lib/jars/config-models/sherpa-config-model-plugin.jar'
[2012-11-27 14:22:50.889] INFO    : configserver     Container.com.yahoo.container.handler.BundleLoader	Installing bundle from disk with reference 'file:/home/vespa/lib/jars/configserver-container-plugin.jar'
[2012-11-27 14:22:54.285] INFO    : configserver     Container.com.yahoo.container.handler.config.HandlersConfigurerDi	Installing bundles from the latest application
[2012-11-27 14:22:57.489] INFO    : configserver     Container.com.yahoo.container.handler.config.HandlersConfigurerDi	Switched to the latest deployed set of handlers, and dependent components, e.g. search chains, searchers and document processors. Application switch number: 0
